src/collections/components/CollectionForm/CollectionForm.tsx

```diff
@@ -199,12 +208,21 @@ export const CollectionForm: React.FC<
     <form name="collection-form" onSubmit={formik.handleSubmit}>
       <Grid container spacing={3}>
         <Grid item xs={12}>
-          <FormikTextField
-            id="title"
-            label="Title"
-            fieldProps={formik.getFieldProps('title')}
-            fieldMeta={formik.getFieldMeta('title')}
-          />
+          <Box display="flex">
+            <Box flexGrow={1} alignSelf="center" textOverflow="ellipsis">
```

Comment:
Do you think it might simplify all these cases to create a component that includes FormikTextField internally?
<FormicCorpusTextField id="title" label="Title" centering_type="box" formic={formic}/> or something like that?

Reply:
Yes, it might be tricky with the css though. Let me merge this for now and do a refactor after.